mobilecoind/src/payments.rs

```diff
@@ -963,8 +962,14 @@ impl<T: BlockchainConnection + UserTxConnection + 'static, FPR: FogPubkeyResolve
         let mut tx_out_to_outlay_index = HashMap::default();
         let mut outlay_confirmation_numbers = Vec::default();
         for (i, outlay) in destinations.iter().enumerate() {
+            // TODO: If you want to support mixed transactions, use outlay-specific token id
+            // here
```

Comment:
Similar comment as: https://github.com/mobilecoinfoundation/mobilecoin/pull/1827/files#r859858717

Also maybe a similar comment as https://github.com/mobilecoinfoundation/mobilecoin/pull/1827/files#r859876530, to have a prioritized checklist of implementing this functionality in clients.